pkgs/games/dwarf-fortress/themes/update.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p jq nix coreutils curl nix-prefetch-git
+
+curl "https://api.github.com/users/dfgraphics/repos" | jq -r '.[].name | ascii_downcase' | while read repo; do
+    version="$(curl "https://api.github.com/repos/DFgraphics/${repo}/releases/latest" | jq -r .tag_name)"
+    sha256="$(nix-prefetch-git "https://github.com/DFgraphics/${repo}" "${version}" | jq -r ".sha256")"
+    echo "{}" | jq ".sha256=\"${sha256}\" | .name=\"${repo}\" | .version=\"${version}\""
```

Comment:
not sure if you can, but do you mind changing the order of version and sha? generally sha comes last with most fetches

Reply:
Do you mean just the key ordering? I actually sort the keys at the end. Is that undesirable?

Reply:
I would say yes, however, it's generated output, so it might not matter so much.

However it's very typical for  fetches to looks like:
```
src = fetchurl {
   name = "...";
   url = "...";
   sha256 = "...";
};
```

Reply:
which for this case I would say should look like:
```
{
  name = "..";
  version = "..";
  sha = "..";
}
```

Reply:
Done, and removed the key sorting.